packages/effect/src/internal/stream.ts

```diff
@@ -4686,6 +4686,39 @@ export const range = (min: number, max: number, chunkSize = DefaultChunkSize): S
     return new StreamImpl(go(min, max, chunkSize))
   })
 
+export const raceAll = <T extends ReadonlyArray<Stream.Stream<any, any, any>>>(
+  ...streams: T
+): Stream.Stream<
+  [T[number]] extends [never] ? never : T[number] extends Stream.Stream<infer A, any, any> ? A : never,
```

Comment:
You can use `Stream.Stream.Success` etc here

Reply:
The funny thing is that it was me who created `Stream.Stream.Success` about a week ago, but i forgot that it was merged:) Definetely it should be used here